Frontend/src/ui/PageTabs.jsx

```diff
@@ -14,15 +14,16 @@ export default function PageTabs() {
   const navigate = useNavigate()
   const location = useLocation()
 
-  const menuItems = useMemo(() => {
+  const [competitorMenu, adminMenu] = useMemo(() => {
```

Comment:
Creating the 2 arrays is unrelated, it would be cleaner to separate them out into 2 definitions.
In fact, the `adminMenu` can probably just be a const outside the component right? It's only shown if `canAdminCompetition`, so the definition can just always be `[registrationsMenuConfig, waitingMenuConfig]`.

Reply:
good point, changed it